### PR TITLE
added a navbar, footer and a page template design

### PR DESCRIPTION
**Pull request by Gmarvis**
image

### CODE REVIEW

File: src/components/organisms/pageTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { ReactNode } from "react";
+import NavBar from "./NavBar";
+import Footer from "./Footer";
+
+const PageTemplate = ({ children }: { children: ReactNode }) => {
```
Comment (RashJrEdmund) [inline src/components/organisms/pageTemplate.tsx:5]: stict to the "function" key word when defining functional components.
```suggestion
function PageTemplate({ children }: { children: ReactNode } ) {
```

**Review by EwiJosepha — APPROVED** [review]

File: src/components/organisms/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import Link from 'next/link'
+import { useState } from 'react'
+
+ const NavBar = () => {
```
Comment (RashJrEdmund) [inline src/components/organisms/NavBar.tsx:4]: "function" key word for functional components
```suggestion
function NavBar() {
```

File: src/components/organisms/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
+ const Footer = () => {
```
Comment (RashJrEdmund) [inline src/components/organisms/Footer.tsx:1]:
```suggestion
function Footer() {
```

File: src/components/organisms/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                    name: 'Team'
+                },
+                {
+                    href: '',
+                    name: 'Careers'
+                },
+            ],
+        }
+    ]
+
```
Comment (RashJrEdmund) [inline src/components/organisms/Footer.tsx:85]: if this has not state mutations, define it out of the component, else momoize it

File: src/components/organisms/pageTemplate.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    return (
+        <main className="">
+            <NavBar />
+            <div className="min-h-screen h-full pt-20 ">{children}</div>
+            <Footer />
+        </main>
+    );
```
Comment (RashJrEdmund) [inline src/components/organisms/pageTemplate.tsx:12]: if you intend to follow semantic html, you `<nav >` and `<footer >` tags should not be inside the `<main>` tag

**Review by RashJrEdmund — CHANGES_REQUESTED** [review]
cool, cool, just fix these small changes 😇

**Pull request merged**